<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupManager</class>
 <widget class="QDialog" name="GroupManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SVAMP: Group Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Ungrouped">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Isolates not in any group</string>
       </property>
       <property name="text">
        <string>Ungrouped</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="ungroupedList">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Select isolates not in any group to add to the selected grroup</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="groupCombo">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Select a group</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_deleteGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete Group</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SubGroup</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="subGroupCombo">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_color">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QListWidget" name="groupList">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Isloates in the selected group</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addSelectedButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Add the selected ungrouped isolates to the group</string>
       </property>
       <property name="text">
        <string>Add Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="removeSelectedButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Remove the selected isolates from this group</string>
       </property>
       <property name="text">
        <string>Remove Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAllButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Remove all of the isloates from this group</string>
       </property>
       <property name="text">
        <string>Remove All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>New Group/Location</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_grname">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Group/Country*</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_latitude">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Latitude</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Longitude</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="newGroupLine">
        <property name="toolTip">
         <string>New group name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_lat"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_lon"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="addGroupButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Add this group to the list</string>
        </property>
        <property name="text">
         <string>Create Group</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="cityLine"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>City/Place*</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>*Mandatory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="doneButton">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Finish and return to the main application</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>177</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newGroupLine</tabstop>
  <tabstop>groupList</tabstop>
  <tabstop>ungroupedList</tabstop>
  <tabstop>removeSelectedButton</tabstop>
  <tabstop>removeAllButton</tabstop>
  <tabstop>addSelectedButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
